src/bundle/Resources/views/themes/admin/ui/component/inputs/input_date_time_picker.html.twig

```diff
@@ -27,7 +27,7 @@
     {% endblock %}
     {% block actions %}
         {{ parent() }}
-        <div class="ibexa-input-text-wrapper__action-btn">
+        <div class="ibexa-input-text-wrapper__action-btn ibexa-input-text-wrapper__action-btn--calendar-icon">
```

Comment:
Just thinking: I think that the initial idea was that it is not a button, but rather a regular icon. In such a case, the problem is not that it's not the button, but that this icon is blocking pointer events on the input. 🤔